app-doc/zeal/zeal-0.7.0_p20231006.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+COMMIT=2cefcd45423e58283372a4c735609c284eded410
```

Comment:
Necessary for what? The alternative would be to intentionally vendor the bugs.

Reply:
Well Its unusual to make a snapshot for seemingly minor issues, usually its for major things  (but in those cases it would be more likely to backport them specially). or if there has been a long time since the last release so they have the fixes have piled up.

As the last release was last month it seems overzealous to already second guess upstream. If they felt it was important for the fixes to be propagated then they would make a release for it.

This is more of value question I guess, there just has to be some balancing about when snapshots are necessary and when not. As otherwise you can go overboard for little reason.

Reply:
I don't know whether it is generally unusual or not, but I don't think this is overboard at the moment and I don't plan to unfix this. I believe this snapshot will most likely result in a better end-user experience than a non-snapshot 0.7.0 ebuild would. You are free to disagree and not merge this. :)

Reply:
Using snapshots is always last resort. If there are important fixes needed, we carry them in FILESDIR.

Reply:
> Using snapshots is always last resort. If there are important fixes needed, we carry them in FILESDIR.

Is this documented somewhere in the Gentoo devmanual or policies?

Ohwell, maybe I'll move them to the `FILESDIR` when I have time.

Reply:
Ok, added the fixes as patches instead.